p_steps/step_12_2_run_scri.R

```diff
@@ -0,0 +1,252 @@
+  # stack and format datasets 
+  output.total <- rbind(output.mod.1, output.mod.2) %>% 
+    mutate(label = case_when(var == "vd1" ~ "pre-exposure pindow", 
+                             var == "vd2" ~ "dose 1 day 0", 
+                             var == "vd3" ~ "dose 1 risk window", 
+                             var == "vd4" ~ "time between dose 1 and dose 2", 
+                             var == "vd5" ~ "dose 2 day 0", 
+                             var == "vd6" ~ "dose 2 risk window", 
+                             TRUE ~ var)) %>% 
+    select(-c(var))
+  
+  # add columns with vaccine and outcome type 
+  # slightly hacky solution here, suggestions for improvement v welcome!
+  output.total$vacctype <- scri_input$vaccinetype[[1]] 
+  output.total$eventtype <- scri_input$eventtype[[1]]
+  
+  output.total <- output.total %>% 
+    relocate(vacctype, eventtype, analysis, label, irr, lci, uci, sei, yi)
+  
+  return(output.total)
+  
+} 
+  
+# Call Functions  ---------------------------------------------------------
+
+# automatically invoke for all datasets 
+file_list <- list.files(path = "./g_intermediate/scri", pattern="input", full.names=TRUE)
+all_results <- map_dfr(file_list, ~run_scri(.x))
+
+# write output
+write.csv(all_results, file = paste0("./g_output/scri/scri_model_output.csv"), row.names=FALSE)
+
+# Checks ------------------------------------------------------------------
+# checks conducted during code development 
+# may be of use for incorporating when testing the function 
+
+# read in one of the dummy case series 
+scri_input <- read.csv("./g_intermediate/scri/AstraZeneca_Myocarditis_input.csv")
+
+# make the data into long format
+# note, the standardscccs function does this implicitly 
+sccs_data_long <- formatdata(indiv = case, 
+                             astart = start, 
+                             aend = end-1,
+                             aevent = event,
+                             adrug = cbind(vd, vd1, vd1+1, vd3, vd2, vd2+1),
+                             aedrug = cbind(vd1-1, vd1, r1end, vd2-1, vd2, r2end), 
+                             dataformat = "multi",
+                             sameexpopar = F, 
+                             data = scri_input) 
+
+# check the first row is always the control period 
+check1 <- sccs_data_long %>% 
+  group_by(indivL) %>% 
+  mutate(counter = row_number()) %>% 
+  filter(vd == 0)
+
+# should be 1 always 
+tabyl(check1$counter)
+
+# check the length of all periods 
+interval_distribution <- sccs_data_long %>%
+  select(vd, interval, event) %>% 
+  tbl_summary(by = vd, 
+              type = all_continuous() ~ "continuous2", 
+              statistic = list(all_continuous() ~ c("{median} ({p25}, {p75})", 
+                                                    "{min}, {max}"),
+                               all_categorical() ~ "{n}")) %>% 
+  modify_header(update = all_stat_cols() ~ "**{level}**")
+
+interval_distribution 
+
```

Comment:
I get another error here:

_Error in modify_header(., update = all_stat_cols() ~ "**{level}**") : 
  could not find function "modify_header"_

Logically, it then also errors because `interval_distribution` does not exist.
This might be because `gtsummary` doesn't load for me. If this also happens on other local places, maybe we should consider an alternative to the `modify_header`?